src/Core/Framework/Api/Controller/InfoController.php

```diff
@@ -296,11 +281,7 @@ private function getBaseUrl(Bundle $bundle): ?string
             return $bundle->getAdminBaseUrl();
         }
 
-        try {
-            if (!$this->filesystem->fileExists(\sprintf('bundles/%s/meteor-app/index.html', mb_strtolower($bundle->getName())))) {
-                return null;
-            }
-        } catch (FilesystemException $e) {
+        if (!file_exists($bundle->getPath() . '/Resources/public/meteor-app/index.html')) {
```

Comment:
here we should also use the Symfony FileSystem. or our own wrapper `\Shopware\Core\Framework\Util\Filesystem`